features/step_definitions/networking.rb

```diff
@@ -1292,3 +1292,60 @@
   default_network = network_operator.default_network(user: admin)
   raise "env doesn't have IPSec enabled" unless default_network["ipsecConfig"]
 end
+
+Given /^I store the hostname from external ids in the#{OPT_SYM} clipboard on the "([^"]*)" node$/ do |cb_ovn_hostname, node_name|
+  ensure_admin_tagged
+  cb_ovn_hostname ||= "ovn_hostname"
+
+  ovsvsctl_cmd = %w(ovs-vsctl list open .)
+  ovn_pod = BushSlicer::Pod.get_labeled("app=ovnkube-node", project: project("openshift-ovn-kubernetes", switch: false), user: admin) { |pod, hash|
```

Comment:
indentation... Also, you can call step %Q/a pod becomes ready with label.../

Reply:
Thanks, fixed indentation. But this step is required for a ready pod on a specific node, looks like %Q/a pod becomes ready with label.../ doesn't have the node specification. So keep the original node here.